application/modules/invoices/models/Mdl_invoices.php

```diff
@@ -591,7 +591,7 @@ public function update_invoice_due_dates($invoice_id)
         $invoice = $this->get_by_id($invoice_id);
         
         if (!empty($invoice)) {
-						$invoice_date_created = date_to_mysql(date(date_format_setting()));          
+            $invoice_date_created = date_to_mysql(date(date_format_setting()));          
```

Comment:
@Verony-makesIT would you mind also removing the extra "enter" on line 619 of this file?

Reply:
I removed the extra "enter" on line 619 as requested.